bin/templates/cordova/lib/check_reqs.js

```diff
@@ -356,12 +364,13 @@ module.exports.check_android_target = function (originalError) {
 module.exports.run = function () {
     return Q.all([this.check_java(), this.check_android()]).then(function (values) {
         console.log('Checking Java JDK and Android SDK versions');
-        console.log('ANDROID_HOME=' + process.env['ANDROID_HOME']);
+        console.log('ANDROID_SDK_ROOT=' + process.env['ANDROID_SDK_ROOT'] + ' (recommended setting)');
+        console.log('ANDROID_HOME=' + process.env['ANDROID_HOME'] + '- DEPRECATED');
 
         if (!String(values[0]).startsWith('1.8.')) {
             throw new CordovaError(
                 'Requirements check failed for JDK 8 (\'1.8.*\')! Detected version: ' + values[0] + '\n' +
-                'Check your JAVA_HOME / ANDROID_HOME / PATH environment variables.'
+                'Check your ANDROID_SDK_ROOT / JAVA_HOME / PATH environment variables.'
```

Comment:
```suggestion
                'Check your `ANDROID_SDK_ROOT` and `JAVA_HOME` PATH environment variables.'
```

Reply:
I am not 100% convinced by this suggestion. It is possible that a developer could set a PATH environment variable that already includes the Android SDK bin directory.

Reply:
ah, then I misread the meaning.

```suggestion
                'Check your `ANDROID_SDK_ROOT`, `JAVA_HOME` and `PATH` environment variables.'
```